### ISSUE TITLE
Allowing clickable footnotes

### ISSUE THREAD

**Issue by CiaranGruber** [source-academy/sicp#867]
As a potential enhancement, it may be better if clicking a footnote in the main section of the text caused a pop-up with the given footnote text. This would help avoid having to go back and forth in the text to check context for the footnote

**Comment by martin-henz** [comment]
Yes, great idea. (Interactive SICP JS is still too much like a physical book; a lot of low-hanging fruit there.)
